src/activity/events.jl

```diff
@@ -124,8 +120,11 @@ function handle_event_request(request, handle;
     return handle(event)
 end
 
-function Base.run(listener::EventListener, args...; kwargs...)
-    return HTTP.serve(listener.server, args...; kwargs...)
+function Base.run(listener::EventListener, host, port, args...; kwargs...)
```

Comment:
This previously used `kwargs` for `port`and `host` because HTTPServer used `kwargs`, while HTTP.jl uses positional args. Could keep them to be backward compatible. Not sure what is best.